.travis.yml

```diff
@@ -126,10 +131,10 @@ install:
     ccache -s
     git describe
     # Upgrade pip and setuptools and wheel to get as clean an install as possible
-    pip install --upgrade pip setuptools wheel
+    python -mpip install --upgrade pip setuptools wheel
```

Comment:
Instead of changing all calls to `pip` to `python -mpip` it would be easier to change all calls to `python` to be `python3` and all calls to `pip` to be `pip3`. That would avoid the need for the symlink.